### ISSUE TITLE
Multiple tabs support

### ISSUE THREAD

**Issue by magicPony** [rationalmatter/Juno-Issue-Tracker#115]
Multiple files opened in different tabs would be an amazing feature I think.

**Comment by navoshta** [comment]
@magicPony Yes, this is certainly on the roadmap! 😉 Can't tell when this will be released exactly, but this will likely arrive in a form of iPadOS's multi-window support.

**Comment by smkirby** [comment]
This would be excellent - I currently find myself (reluctantly) switching back to using safari for Jupyter notebooks because I often need to see two notebooks side by side (although even switching quickly between them in full screen would help).

**Comment by harveydov** [comment]
This would be a huge one for me. Look forward to this. Keep up the good work. Thanks
